arlo.code-workspace

```diff
@@ -7,5 +7,13 @@
 			"path": "arlo-client"
 		}
 	],
-	"settings": {}
+	"settings": {
+		"remote.SSH.defaultForwardedPorts": [
```

Comment:
I'm not versed in docker well enough, but I'm assuming this means ssh tunneling? Is that the standard way to do it?

Reply:
> I'm assuming this means ssh tunneling?

Yes, it's SSH port forwarding / tunneling. This actually has nothing to do with Docker. This is for when you're doing [remote development with SSH](https://code.visualstudio.com/docs/remote/ssh), which is what I use.